pct.sh

```diff
@@ -20,6 +20,12 @@ if ! [[ $PLUGINS =~ blueocean || $PLUGINS =~ lockable-resources || $PLUGINS =~ p
 	PCT_D_ARGS+='-DforkCount=.75C '
 fi
 
+# Tracked by .github/renovate.json
+JTH_VERSION=2411.v1e79b_0dc94b_7
```

Comment:
Intentionally one version behind to test that the automatic update works after this PR is merged.